index.js

```diff
@@ -12,7 +14,10 @@ function getAssignmentTypes(assignments) {}
  * @param {number} score.max - The maximum amount of points that could be received on the assignment.
  * @returns {number} A rounded integer, representing a percentage.
  */
-function getPercentageScore(score) {}
+function getPercentageScore(score) {
+  let percent = (score.received / score.max) * 100;
```

Comment:
You don't _need_ the `percent` variable.  Though you're certainly welcome to prefer breaking it down in the end, try making this all one line for an exercise in concise coding.